strawberry/field.py

```diff
@@ -9,13 +9,15 @@
 from .utils.str_converters import to_camel_case
 
 
-def check_return_annotation(field_definition: FieldDefinition):
-    f = cast(Callable, field_definition.base_resolver)
-    name = cast(str, field_definition.name)
+def get_return_annotation(field_definition: FieldDefinition) -> Type:
+    resolver = cast(Callable, field_definition.base_resolver)
+    name = resolver.__name__
```

Comment:
Discussion: This is no longer the camelcase-d resolver name. I figure this is actually preferable, as this error will occur at startup. The camelcase name would only be helpful if this was a runtime error